### ISSUE TITLE
Configure APIGateway from Swagger files?

### ISSUE THREAD

**Issue by aletheia** [jaws-framework/JAWS#283]
Hi, we're starting to use JAWS and are very excited about it. 
One of our biggest concerns is about being able to use our Swagger API description files as source for APIGateway configuration (maybe through swagger importer).
Having not to write the CloudFormation-like configuration and being able to share API description files with out customer would be really great.

**Comment by marklawlor** [comment]
I'm working on a JAWS-Swagger integration plugin, its still in development but should be finished Soon™

https://github.com/marklawlor/jaws-swagger

**Comment by aletheia** [comment]
Supercool!!
Is there any way I could help you? My company is very interested in adopting JAWS for their microservices, so I have some time to be spent doing R&D on this project. If this could be helpful I would be glad of it.

**Comment by marklawlor** [comment]
Give me 24hours to update the repo / add a contributing guide. I'll be adding a roadmap as well for people to help contribute.

**Comment by doapp-ryanp** [comment]
This function already exists via https://github.com/awslabs/aws-apigateway-importer

**Comment by aletheia** [comment]
Yeag, we're using it but it is NOT used by JAWS, to my understanding. This
means we have to write configuration twice or mantain a link between
different conf file.
This means also having a manual step when deploying

What am I missing?
Il giorno mer 4 nov 2015 alle 04:02 Ryan Pendergast <
notifications@github.com> ha scritto:
